### PR TITLE
Loading Loader

### PR DESCRIPTION
**Pull request by eric-gade**
## What does this PR do? 🛠️
This PR implements the design and specification in #1249. Specifically, we add a "loader" image and placeholder text when the user has selected a new location from the search and the page has started the loading process.

## What does the reviewer need to know? 🤔
I have added a Playwright test to ensure the loader displays. Because of that, I had to convert the whole combo-box test file.

## Screenshots (if appropriate): 📸
<details>
<summary>Point page (desktop):</summary>

Screenshot 2024-06-04 at 6 48 15 AM
</details>
<details>
<summary>Point page (mobile):</summary>

Screenshot 2024-06-04 at 6 48 08 AM
</details>
<details>
<summary>Front page (mobile):</summary>

Screenshot 2024-06-04 at 6 48 33 AM
</details>
<details>
<summary>Front page (desktop):</summary>

Screenshot 2024-06-04 at 6 48 24 AM
</details>

### CODE REVIEW

File: web/themes/new_weather_theme/assets/sass/components/_wx-loader.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  background-image: url("../images/weather/wx_loader_animated_cloud.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+@media (prefers-reduced-motion){
+  .wx-loader-icon {
+    background-image: url("../images/weather/wx_loader_static.svg");
+  }
+}
```
Comment (greg-does-weather) [inline web/themes/new_weather_theme/assets/sass/components/_wx-loader.scss:13]: This is lovely. Just lovely.

**Review by greg-does-weather — CHANGES_REQUESTED** [review]
Code looks good. I'm amused/annoyed by those two rando test failures, each affecting a different browser. Probably just timing out in the workflow.

- One comment below about making the loader also show up when users select a location based on their previous location pages.
- The loader also doesn't show up if the user clicks the "use my location" link/button.

Once those are resolved, I think this will be good to go!

**Review by partly-igor — APPROVED** [review]
Design is looking good!

**Review by greg-does-weather — APPROVED** [review]
loading

**Pull request merged**